api/apps/geoprocessing/src/legacy-project-import/legacy-piece-importers/file-readers/dat-file.comma-or-tab-finder.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { Test } from '@nestjs/testing';
+import { isRight, isLeft } from 'fp-ts/lib/Either';
+import { Readable } from 'stream';
+import {
+  DatFileCommaOrTab,
+  DatFileCommaOrTabFinder,
+} from './dat-file.comma-or-tab-finder';
+import { invalidDelimiter } from './dat-file.delimiter-finder';
+
+let fixtures: FixtureType<typeof getFixtures>;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+});
+
+it('fails when dat file uses invalid delimiter', async () => {
+  const fileLocation = fixtures.GivenADatFileWithInvalidDelimiter();
+  const result = await fixtures.WhenGettingDaDelimiter(fileLocation);
```

Comment:
tiny typo - I suppose you meant `WhenGettingDatDelimiter` (here and in other tests + fixtures)